packages/expo-modules-core/android/build.gradle

```diff
@@ -0,0 +1,46 @@
+apply plugin: 'com.android.library'
```

Comment:
I believe we need to add kotlin to this module: https://github.com/expo/expo/pull/12716